salt/files/minion.d/f_defaults.conf

```diff
@@ -43,6 +43,12 @@ master:
 # choose a random master
 {{ get_config('random_master', 'False') }}
 
+# Set the master type
+{{ get_config('master_type', 'str') }}
```

Comment:
And now it appears as if they have changed it in 2014.7 and 2015.5. That's going to severely complicate things.

https://github.com/saltstack/salt/commit/cc514938a86a4cdab93ff0ee69d5d47f562ddc45